docs/NMOS With JPEG XS.md

```diff
@@ -67,10 +67,13 @@ An example Flow resource is provided in the [Examples](../examples/).
 
 ### Senders
 
-The Sender resource MUST indicate `urn:x-nmos:transport:rtp` or one of its subclassifications for the `transport` attribute.
+For Nodes transmitting JPEG XS using the RTP payload mapping defined by RFC 9134, the Sender resource MUST indicate `urn:x-nmos:transport:rtp` or one of its subclassifications for the `transport` attribute.
 Sender resources provide no indication of media type or format, since this is described by the associated Flow resource.
 
 The SDP file at the `manifest_href` MUST comply with the requirements of RFC 9134.
+Additionally, the format-specific parameters defined by RFC 9134 are made REQUIRED in the same circumstances that a corresponding Flow attribute MUST be included, in order that the information provided to an NMOS Controller through a Flow is also provided to a Receiver through the SDP file.
+This means that `profile`, `level`, `sublevel`, `sampling`, `depth`, `width`, `height`, `exactframerate`, `colorimetry`, `interlace` and `segmented` are all REQUIRED depending on the properties of the JPEG XS stream.
```

Comment:
Will try spelling this out to make it clearer in what circumstances `profile`, `level` and `sublevel` MUST be omitted, and the same for `interlace` and `segmented`.

Reply:
> the only advantage of putting them into the SDP would be to allow a receiver to reject the inappropriate stream without joining it.

I think your assessment is accurate, John. How useful is this? It enables a Receiver to produce a meaningful error status for both immediate activations and scheduled activations. If that's the only advantage is that worthwhile?
How onerous is this for Senders if they are already required to have this information in the NMOS resources? That may depend on whether the NMOS Node is a proxy for a device that produces the SDP file.

Reply:
Discussed in activity group call yesterday.

> the only advantage of putting them into the SDP would be to allow a receiver to reject the inappropriate stream without joining it.

@David-P-B made the point that if there is sufficient information in the SDP data the Receiver can configure a decoder before joining the stream. @jmailhot showed that TR-08 includes all the optional parameters in its example (apart from `profile` which @tbr thought was an oversight), which we take as best practice.

Reply:
Originally I believe `profile` was omitted because TR-08 only used the High444.12 profile. However, for 4:2:0 another profile must be used, hence TR-08 allows either `High444.12` or `High420.12`. I believe we forgot to update the SDP example (in TR-08).